lnbits/wallets/base.py

```diff
@@ -18,6 +18,18 @@ class InvoiceResponse(NamedTuple):
     payment_request: Optional[str] = None
     error_message: Optional[str] = None
 
+    @property
+    def success(self) -> bool:
+        return self.ok is True
+
+    @property
+    def pending(self) -> bool:
+        return self.ok is None
+
+    @property
+    def failed(self) -> bool:
+        return self.ok is False
+
```

Comment:
hmm. just want to say it here i am not a huge fan of this magic properties, and also that `ok` has 3 states, True, None, False
ignore this comment